modules/ROOT/pages/cloudhub-networking-guide.adoc

```diff
@@ -103,21 +109,21 @@ Depending on what region you deploy your application in, the DNS record and the
 
 |===
 
-Deploying to a region also affects your internal and external worker DNS address. For example, if you deploy in Europe, the DNS records for internal and external IPs are `mule-worker-myapp.eu.cloudhub.io` and `mule-worker-internal-myapp.eu.cloudhub.io`.
+Deploying to a region also affects your internal and external worker DNS address. For example, if you deploy in Europe, the DNS records for internal and external IP addresses are `mule-worker-myapp.eu.cloudhub.io` and `mule-worker-internal-myapp.eu.cloudhub.io`.
 
 == Firewall Rules and Port Access
 
-The only two ports exposed externally are `${http.port}` and `${https.port}` (by default 8081 and 8082 respectively). If you wish to access other ports, you can do so through the xref:virtual-private-cloud.adoc[*Anypoint Virtual Private Cloud (Anypoint VPC)*] offering.
+The only two ports exposed externally are `${http.port}` and `${https.port}` (by default 8081 and 8082 respectively). If you want to access other ports, you can do so through the xref:virtual-private-cloud.adoc[*Anypoint Virtual Private Cloud (Anypoint VPC)*] offering.
```

Comment:
Another use of bold for emphasis: [*Anypoint Virtual Private Cloud (Anypoint VPC)*] -- remove the stars